templates/components/api-server/component.js

```diff
@@ -16,10 +16,9 @@ template.package = {
     "loopback": "^2.22.0",
     "loopback-boot": "^2.6.5",
     "loopback-datasource-juggler": "^2.39.0",
-    "serve-favicon": "^2.0.1"
-  },
-  "optionalDependencies": {
-    "loopback-explorer": "^1.1.0"
+    "loopback-component-explorer": "^2.1.0",
+    "serve-favicon": "^2.0.1",
+    "serve-static": "^1.10.0"
```

Comment:
yes i know, but i the example has these new lines in it:

```
"files": {
    "serve-static": {
      "params": "$!../client"
    }
  },
```

https://github.com/strongloop/loopback-example-app/blob/master/server/middleware.json#L26